app/assets/javascripts/covid19/src/webapp/components/structures-table/Toolbar.tsx

```diff
@@ -15,6 +15,16 @@ import { HtmlTooltip } from "./HtmlTooltip";
 import i18n from "../../../utils/i18n";
 import "./Toolbar.css";
 
+export const searchExamples = [
+    "EMD-21375",
+    "Glycoprotein",
+    "Remdesivir",
+    "3CL-Pro",
+    "Mouse",
+    "PanDDA",
+    "Martinez",
+];
```

Comment:
Some words are lowercase in the task (ie. "mouse"), I guess it's not that important.

Reply:
Search is case insensitive, so I turned search examples uppercase for better looking. Should I follow more explicitly what has been asked? @adrianq

Reply:
Yes, please, in this case, in particular, I think it is important to follow the client requirements as they have been requested.